product_pack/views/product_product_views.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <record model="ir.ui.view" id="nan_product_pack_form">
+        <field name="name">product.product.pack.form</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_normal_form_view" />
+        <field name="arch" type="xml">
+            <field name="type" position="after">
+                <field name="pack"/>
+                <field name="pack_type" attrs="{'invisible':[('pack', '=', False)], 'required':[('pack','=',True)]}" context="{'pack_type': pack_type}"/>
```

Comment:
IMHO it would make more sense for the user if you put these 2 fields into the notebook page you create a few lines below....

Reply:
@ernestotejeda @Yajo @rafaelbn 

I am agreed with @rafaelbn in this case

IMHO I think will be better to have the "Pack" boolean field as checkbox along side with "​Can be Purchased" and "Can be Sold". And if this field is show/hide the "Pack" page.

This way we can put all the pack related fields inside the "Pack" page and we do not need to apply invisible or not attributes by field.

What do you think?

Reply:
Hmm... ok, it makes sense too. I gues the bool field should be named `pack_ok` for consistency.

Reply:
I agree with @zaoral too, since `"Pack?"` field will determine the visibility of the "Pack" page that is inside the notebook just like "Can be Purchased" and "Can be Sold" with their corresponding pages.
And I agree with @Jairo to name this field pack_ok

Reply:
Perfect, @ernestotejeda please could you implement this change and let us know.
change the field to pack_ok LGTM